src/demo/contexts/demo/defaultImages.ts

```diff
@@ -1,129 +1,7 @@
+import dataset from '../../../../static/demoImages/imageData.json';
```

Comment:
You put this in static as if the resources needed to be fetchable, but it does not. It can be colocated with where we need it. It will be bundled with the js files.